src/Microsoft.Framework.PackageManager/Utils/PackageSourceUtils.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text.RegularExpressions;
```

Comment:
Nope! I started with a regex then realized that was super overkill. I'll remove it.